examples/with-three-js/next.config.js

```diff
@@ -0,0 +1,4 @@
+const withCSS = require('@zeit/next-css')
```

Comment:
We're no longer accepting examples that use `@zeit/next-css`. Please upgrade to the build-in CSS support.

Reply:
Done

Reply:
@afsanefda Next.js supports importing CSS files now. You did not need to switch to styled-jsx!

Reply:
I thought you meant styles-jsx! I can rollback it to the css file if it is possible.

Reply:
@Timer Done

Reply:
@afsanefda `next-css` is still in use, can you switch to CSS Modules?

Reply:
to avoid confusion, CSS Modules and importing global styles in _app.js is currently our built-in CSS Support, and you won't need `@zeit/next-cs` for that, so this file can be removed.

Reply:
> to avoid confusion, CSS Modules and importing global styles in _app.js is currently our built-in CSS Support, and you won't need `@zeit/next-cs` for that, so this file can be removed.

Done! I've added _app.js and global built-in CSS  is replaced with the @zeit/next-css!